backend/app/services/search.py

```diff
@@ -313,6 +314,126 @@ async def get_indices_from_alias(self, es_client, alias):
             print(f"Error retrieving indices for alias '{alias}': {e}")
             return []
 
+    async def buildFilterData(self, filter_, total):
```

Comment:
I commented on this elsewhere, but ... why `filter_`? The underscore is easy to miss in some fonts, and seems to have no real significance here.

Reply:
This has been updated in the PR #150